packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -260,36 +260,9 @@ export const Combobox = <T extends string | number>({
       : 'angle-down';
 
   return (
-    <div>
+    <Box position="relative">
       <InputComponent
         width={width === 'auto' ? undefined : width}
-        suffix={
```

Comment:
Is this something to be fixed in the `InputComponent` itself?

Reply:
I think that would be acceptable. In cases where it is not desired, the consumers could always to `event.stopPropagation()`